tooling/profiler/src/flamegraph.rs

```diff
@@ -82,25 +130,26 @@ impl FlamegraphGenerator for InfernoFlamegraphGenerator {
     }
 }
 
-fn generate_folded_sorted_lines<'files, F: AcirField>(
-    samples: Vec<Sample<F>>,
+fn generate_folded_sorted_lines<'files, S: Sample>(
+    samples: Vec<S>,
     debug_symbols: &DebugInfo,
     files: &'files impl Files<'files, FileId = fm::FileId>,
 ) -> Vec<String> {
+    println!("About to generate sorted lines");
```

Comment:
```suggestion
```

Reply:
Noticed we actually have a decent number of print statements in here so maybe this is on purpose.

Reply:
The other ones are on purpose, but this one is leftover from debugging.